packages/terra-consumer-icon/scripts/src/generate-icon/template.txt

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import React from 'react';
+import IconBase from '../../node_modules/terra-icon/lib/IconBase';
```

Comment:
I tried this but we are not able to import. So as of now only way to import icon and other component is to get them from node/modules/terra-icon/lib ....